src/content/en/updates/2017/10/devtools-release-notes.md

```diff
@@ -29,7 +29,7 @@ Note: You can check what version of Chrome you're running at
 `chrome://version`. Chrome auto-updates to a new major version about every 6
 weeks.
 
-## Multi-client remote debugging support {: #multi-client }
+## Multi-client debugging support {: #multi-client }
```

Comment:
We described it as "Multi-client remote debugging support" in the UI

https://chromium-review.googlesource.com/c/chromium/src/+/713460/6/third_party/WebKit/Source/devtools/front_end/help/ReleaseNoteText.js#19